toolkit/tools/pkg/imagecustomizerlib/customizeutils.go

```diff
@@ -89,6 +91,12 @@ func doCustomizations(buildDir string, baseConfigPath string, config *imagecusto
 		return fmt.Errorf("failed to add extra kernel command line: %w", err)
 	}
 
+	err = handleSELinux(config.SystemConfig.KernelCommandLine.SELinux,
```

Comment:
should selinux `setfiles` be called as the last step after running scripts?

Reply:
No, this is actually one of the meaningful differences between PostInstallScripts and FinalizeImageScripts, and mirrors the imager's behavior.